Scripts/build.cake

```diff
@@ -0,0 +1,159 @@
+#tool "nuget:?package=xunit.runner.console"
```

Comment:
I believe the build scripts coming from DEVOPS-440 shouldn't be part of this PR. can you transfer your changes to a feature branch and submit a new PR? thx